single-hit-redirect/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Single Hit Redirect Demo</title>
+    <link rel="stylesheet" href="../demo-styles.css">
+
+    <script src="../bower_components/ff-web-components/dist/vendor/custom-elements-es5-adapter.js"></script>
+    <script src="../bower_components/ff-web-components/dist/vendor/webcomponents-loader.js"></script>
+    <script defer src="../bower_components/ff-web-components/dist/bundle.js"></script>
+    <script>
+        document.addEventListener("WebComponentsReady", function () {
+            factfinder.communication.ResultDispatcher.subscribe('result', function(result) {
+                if (!+factfinder.common.localStorage.getItem('ff_redirect_off') || factfinder.common.localStorage.getItem('ff_redirect_referer') !== window.location.href) {
```

Comment:
Is `!+` a typo? Other than that I would extract `factfinder.common.localStorage` for readability.

Reply:
Looks like a dirty string to int conversion. I'd avoid such hacks and use strings everywhere for readability.

Reply:
@KrzysztofKozlowskiOBSS  changed, didn't knew using + operator to convert value to number is considered as a hack

@fmarangi  by extract you mean something like this?  
` const storage = factfinder.common.localStorage;`